packages/website/src/icons/CssIcon.tsx

```diff
@@ -0,0 +1,33 @@
+import { Props } from './types';
+
+export function CssIcon({ size = 800, className }: Props) {
+  return (
+    <svg
+      width={`${size}px`}
+      height={`${size}px`}
```

Comment:
I believe `px` is not necessary here. 
```suggestion
      width={size}
      height={size}
```